src/EnergyPlus/DXCoils.cc

```diff
@@ -10314,10 +10316,12 @@ namespace DXCoils {
         // Get condenser outdoor node info from DX Heating Coil
         if (DXCoil(DXCoilNum).CondenserInletNodeNum(1) != 0) {
             OutdoorDryBulb = Node(DXCoil(DXCoilNum).CondenserInletNodeNum(1)).Temp;
+            CompAmbTemp = OutdoorDryBulb;
             if (DXCoil(DXCoilNum).CondenserType(Mode) == WaterCooled) {
                 OutdoorHumRat = OutHumRat;
                 OutdoorPressure = OutBaroPress;
                 OutdoorWetBulb = OutWetBulbTemp;
+                CompAmbTemp = OutDryBulbTemp;
```

Comment:
This seems to be the one outlier here, while everyone else is OutdoorDryBulb, this is OutDryBulbTemp.  I'm not sure I understand why that is yet, but I'll keep reading along.

Reply:
There are 2 indicators here, one is local environment around the compressor (for e.g., crankcase heater) and the other is condenser inlet fluid temp. I'm not sure I have this completely correct but these changes were a big move in the right direction.